composer.json

```diff
@@ -20,13 +20,14 @@
     "require": {
         "php":                  "^7.4.0 || ^8.0",
         "doctrine/orm":         "^2.9.5",
-        "doctrine/persistence": "^2.2.2"
+        "doctrine/persistence": "^2.2.2 || ^3.0.2"
     },
     "require-dev": {
         "doctrine/coding-standard": "^9.0.0",
```

Comment:
I think we need `ext-pdo_sqlite: *` in the dependencies: it's used in the tests, but that's not made clear here